code/modules/clothing/suits/armor.dm

```diff
@@ -519,6 +527,8 @@
 		/obj/item/tank/internals/emergency_oxygen,
 		/obj/item/tank/internals/plasmaman,
 		)
+/obj/item/clothing/suit/armor/riot/init_rustle_component
```

Comment:
buddy I love you but this line ismissing a key component

Reply:
oh it was supposed to be:
```php
/obj/item/clothing/suit/armor/riot/knight/init_rustle_component
	return
```